content/spinup.md

```diff
@@ -0,0 +1,133 @@
+# Setup CTSM
+
+For running CTSM on the Norwegian cluster you will be affilicated with a project account. 
+
+To check which project you are affiliated with can run by typing
+
+    [~/HOME]$ projects
```

Comment:
If `HOME` is the home directory environment variable, then should we have `~/$HOME`?

(related to https://github.com/MetOs-UiO/CTSM-Norway-Documentation/issues/3)

Reply:
Related to #7 .

Let's do a total review on the point when the total version 1 of the documentation is ready.